mungegithub/mungers/e2e/e2e.go

```diff
@@ -264,7 +266,10 @@ func (e *RealE2ETester) LatestRunOfJob(jobName string) (int, error) {
 
 // LoadNonBlockingStatus gets the build stability status for all the NonBlockingJobNames.
 func (e *RealE2ETester) LoadNonBlockingStatus() {
-	for _, job := range e.NonBlockingJobNames {
+	e.Opts.Lock()
+	jobs := *e.NonBlockingJobNames
```

Comment:
For safety, all these uses should probably copy the string slices, to avoid weird races.